src/topology.js

```diff
@@ -155,7 +155,11 @@ Topology.prototype.configureExchanges = function( exchangeDef, list ) {
   }
 };
 
-Topology.prototype.createBinding = function( options ) {
+Topology.prototype._binding = function ( options, isCreateOperation ) {
+
+  const queueOperation = isCreateOperation ? "bindQueue" : "unbindQueue";
+  const ExchangeOperation = isCreateOperation ? "bindExchange" : "unbindExchange";
```

Comment:
`const ExchangeOperation` starts with capital letter switch to `const exchangeOperation`

Reply:
It's fixed: 4eccb5567d99da18a3ff59f8831ba1f16913bd3c